lib/matplotlib/projections/polar.py

```diff
@@ -995,15 +995,43 @@ def _gen_axes_spines(self):
         return spines
 
     def set_thetamax(self, thetamax):
+        """
+        Set the maximum theta limit.
+
+        Parameters
+        ----------
+        thetamax : float
+            Maximum theta limit in degrees.
+        """
         self.viewLim.x1 = np.deg2rad(thetamax)
 
     def get_thetamax(self):
+        """
+        Returns
```

Comment:
Docstrings should always have a short summary.

You can shorten all the docstrings in the PR to the pattern:
~~~
"""Return the maximum theta limit in degrees."""
~~~